static/theme-flex/style.css

```diff
@@ -2692,7 +2692,7 @@ a[href*="#no-click"], img[src*="#no-click"] {
     article section.page p>code,
     article section.page li code,
     article section.page table code {
-        display: block;
+        /* display: block; */
```

Comment:
This was making inline code text move to its own line. You can see it at `/rs/installing-upgrading/downloading-installing/#prerequisites` in the expanding block.
Is it safe to make this change?

Reply:
I checked it, the change is safe to make.